maxim/logger/fireworks/helpers.py

```
@@ -0,0 +1,171 @@
+
+        except Exception as e:
+            scribe().warning(
+                f"[MaximSDK][FireworksInstrumentation] Error in logging generation: {e}",
+            )
+    
+    @staticmethod
+    async def async_stream_helper(
+        stream: AsyncGenerator[ChatCompletionChunk, None], 
+        generation: Generation, 
+        trace: Trace, 
+        is_local_trace: bool
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)